assignments/mini-libc/samples/Makefile

```diff
@@ -0,0 +1,27 @@
+LIBC_PATH ?= ../libc
+CPPFLAGS = -nostdinc -I. -I$(LIBC_PATH)/include
+CFLAGS = -Wall -Wextra -fno-PIC -fno-stack-protector -fno-builtin
```

Comment:
Add right below this line debugging flags:
`# CFLAGS += -g -O0 -std=c99 # For debugging.`